libs/sdk-core/src/greenlight/node_api.rs

```diff
@@ -708,6 +708,25 @@ impl NodeAPI for Greenlight {
         Ok(res.bolt11)
     }
 
+    async fn lookup_bolt11(&self, payment_hash: Vec<u8>) -> NodeResult<Option<String>> {
```

Comment:
I think `lookup_bolt11` implies retrieving the bolt11 from local persistence. Maybe `fetch_bolt11` describes it better and seems to be more inline with other functions here.

Reply:
Done: 31e56b12870421103bde7354783d7c4365b18f94